### ISSUE TITLE
Errors raised when loading Tiff file

### ISSUE THREAD

**Issue by AuSpotter** [rnd-team-dev/plotoptix#29]
I have a tiff file, which displays correctly but also leads to many errors popping up: 

`ReadDirectory: Warning, C:\Users\image.tif: unknown field with tag 33550 (0x830e) encountered`

Is this because I'm using geotiff RGBA files?

**Comment by robertsulej** [comment]
Hi Shawn,

I saw this message when reading tiff files. It comes from the libtiff library, which is used to load tiff format. In my case it did not block reading the image. I guess libtiff does not recognize all the content possible to store in tiff.

Does your image load anyway? If you can attach sample image I can have a look at the problem and check.

I'll also check if there are any updates to libtiff.

**Comment by AuSpotter** [comment]
Yes, I was also able to load and display the image without issue!
